script/appveyor/cuda_install.cmd

```diff
@@ -7,7 +7,9 @@ cuda_8.0.44_windows.exe -s compiler_8.0 ^
                            cublas_dev_8.0 ^
                            cudart_8.0 ^
                            curand_8.0 ^
-                           curand_dev_8.0
+                           curand_dev_8.0 ^
+                           cufft_8.0 ^
+                           cufft_dev_8.0
```

Comment:
I'm just thinking do we really need normal and **dev** versions of the same package? What do you think?

Reply:
Hmm, it looks like you're right. At least for cuda-cufft dev version depends on the normal version and they are both going to be installed. So we need only dev package.